src/utils/index.test.js

```diff
@@ -1,5 +1,83 @@
-const { isMaxGuesses } = require('./utils');
+const { EvaluationColor, getEvaluation, isMaxGuesses, isWin } = require('../utils');
 
-test(`test test`, () => {
-    expect(isMaxGuesses({})).toBe(false);
+/*
+ * getEvaluation
+ */
+
+test('evaluations all match', () => {
+    const word = "abcde";
+    const guess = "abcde";
+    const expected = [EvaluationColor.Green, EvaluationColor.Green, EvaluationColor.Green, EvaluationColor.Green, EvaluationColor.Green];
+    expect(getEvaluation(word, guess)).toEqual(expected);
+})
+
+test('no evaluations match', () => {
+    const word = "abcde";
+    const guess = "fffff";
+    const expected = [EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray];
+    expect(getEvaluation(word, guess)).toEqual(expected);
+})
+
+test('one match', () => {
+    const word = "abcde";
+    const guess = "aaaaa";
+    const expected = [EvaluationColor.Green, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray];
+    expect(getEvaluation(word, guess)).toEqual(expected);
+})
+
+test('one partial', () => {
+    const word = "abcde";
+    const guess = "faaaa";
+    const expected = [EvaluationColor.Gray, EvaluationColor.Yellow, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray];
+    expect(getEvaluation(word, guess)).toEqual(expected);
+})
+
+test('one match, one partial', () => {
+    const word = "abbde";
+    const guess = "aaaba";
+    const expected = [EvaluationColor.Green, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Yellow, EvaluationColor.Gray];
+    expect(getEvaluation(word, guess)).toEqual(expected);
+})
+
+test('one match, no yellow overwrite', () => {
+    const word = "abbbb";
+    const guess = "aaccc";
+    const expected = [EvaluationColor.Green, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray];
+    expect(getEvaluation(word, guess)).toEqual(expected);
+})
+
+/*
+ * isMaxGuesses
+ */
+
+test(`not at max guesses`, () => {
```

Comment:
I generally find myself only using template strings when I'm doing interpolation, otherwise I just use normal quotes.  I can't find anything that gives a programatic reason not to use them for normal basic literal strings, just convention.

Reply:
Oh I didn't even realize I was using those I must have copy pasted and not noticed. Great catch!